data/table.txt

```diff
@@ -7671,12 +7707,14 @@
 儭 儭 1 1 0 0 1 0 0 0 0 oydu oydu NA 4615
 儮 儮 1 1 0 0 1 0 0 0 0 omdm omhm NA 4614
 儯 儯 1 0 0 0 1 0 0 0 0 ofqf ofqf NA 0
-儰 儰 1 1 0 0 1 0 0 0 0 otif otbf,otif NA 7383
+儰 儰 1 1 0 0 1 0 0 0 0 otif otbf NA 7383
+儰 儰 1 1 0 0 1 0 0 0 0 NA otif NA 0
 儱 儱 1 1 0 0 1 0 0 0 0 oybp oybp NA 4616
 儲 储 1 1 0 0 1 0 0 0 0 oyra oyra NA 16006
 儳 儳 1 1 0 0 1 0 0 0 0 onri onri NA 15263
 儴 儴 1 1 0 0 1 0 0 0 0 oyrv oyrv NA 4052
-儵 儵 1 1 0 0 1 0 0 0 0 olof olhf,olof,xxolh,xxolo NA 4051
+儵 儵 1 1 0 0 1 0 0 0 0 olof olhf,xxolh NA 4051
+儵 儵 1 1 0 0 1 0 0 0 0 NA olof,xxolo NA 0
```

Comment:
@yookoala [said](https://github.com/Cangjians/libcangjie/commit/d9d2e686ce6776ec67e89b4256dce42359597d84#r29099611):

> "olof" and "xxolo" seems to be standard mapping on MacOS.

---

Ok, so this is the one which should have the non-0 frequency?

It got 0 from my script because it was ordered last:

* the character had `olhf,olof,xxolh,xxolo` as CJ5 codes
* my script grouped the x-disambiguation with the canonical code, leading to two groups in the same order they were specified originally: `olhf,xxolh` and `olof,xxolo`
* the script gave 0 as frequency for the second group, as you had suggested in #104 

I guess what you're saying is that the table had a wrong order then, and `olof` should have been before `olhf`, at which point my script would have given `oloh` a frequency of 4051 and `olhf` a frequency of 0?

Reply:
So Wan Leung just said that Mac OS X uses the short-lived Cangjie version 4, not version 5. :open_mouth:

Discussing this with @yookoala, we decided to ignore all those comments about Mac.